mozilla/fx-private-relay
# MPP-3475 - String updates for profile page

This PR fixes MPP-3475

L10N: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/160 
 


# New feature description
**Acceptance Criteria**


- Add indication on the mask if any blocking (promo and all) is applied or are forwarding all emails
- Add updated content about blocking promo emails on the detail of the mask
- Content and UI looks like [Block promotions selected Figma](https://www.figma.com/file/zA6iEpjHkLYeloGQSHZSfX/Relay-Custom-Domain-Management?type=design&node-id=332-3178&mode=design&t=PRflm6ivfLrKO0zU-4)
- Content and UI looks like [Premium dashboard Figma](https://www.figma.com/file/zA6iEpjHkLYeloGQSHZSfX/Relay-Custom-Domain-Management?type=design&node-id=332-2837&mode=design&t=PRflm6ivfLrKO0zU-4)
- While working on frontend, please check [mobile Figma](https://www.figma.com/file/zA6iEpjHkLYeloGQSHZSfX/Relay-Custom-Domain-Management?type=design&node-id=352-14006&mode=design&t=KHmmCcMHQlwIZWRe-4) to double confirm the UX looks correct on mobile as well.

# Screenshot (if applicable)

![image](https://github.com/mozilla/fx-private-relay/assets/3924990/472adabe-4688-48a8-94c0-9b312e117c35)


https://github.com/mozilla/fx-private-relay/assets/3924990/5e6192b0-5f89-41d1-bf48-febc25283c74



# How to test

Visit the profile page of a free user and premium user to verify new strings. Changing the blocking state for a mask will update the text shown on the card itself (top right) to reflect what the mask is doing. 

# Checklist (Definition of Done)
